contracts/PurchasabilityValidator.sol

```diff
@@ -1,13 +1,76 @@
 pragma solidity 0.5.7;
+pragma experimental ABIEncoderV2;
 
 import "./DrugItem.sol";
 
 contract PurchasabilityValidator {
 
     uint8 public VALID_FOR_PURCHASE = 100;
+    uint8 public NOT_IN_PHARMACY = 200;
+    uint8 public TOO_MANY_HANDOVERS = 201;
+    uint8 public TEMPERATURE_TOO_HIGH = 202;
+    uint8 public TEMPERATURE_TOO_LOW = 203;
+    uint8 public TOTAL_TRANSIT_TIME_TOO_LONG = 204;
+    uint8 public SINGLE_TRANSIT_TIME_TOO_LONG = 205;
 
     function isPurchasable(DrugItem _drugItem) public view returns (uint8[] memory result) {
```

Comment:
This method is definitely too long and hard to read split to smaller private methods